CHANGES.rst

```diff
@@ -423,4 +426,4 @@ CHANGES
 0.3.0
 -----
 
-* Added ``QueryManager``
+- Added ``QueryManager``
```

Comment:
This is up to you, I can also use `* ` everywhere. Just a matter of taste but I feel that we shouldn't mix them.